src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -608,6 +608,17 @@ protected override void BeginProcessing()
                     ThrowTerminatingError(errorRecord);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.https://examples.oreilly.com/9780596528492/ should work
```powershell
retry of URL failed with error: Connection timed out
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```